q2_sample_classifier/_transformer.py

```diff
@@ -65,13 +66,14 @@ def _4(data: pd.Series) -> (PredictionsFormat):
 def _5(ff: PredictionsFormat) -> (pd.Series):
     with ff.open() as fh:
         df = _read_dataframe(fh)
-        return df.iloc[:, 0]
+        return pd.to_numeric(df.iloc[:, 0], errors='ignore')
```

Comment:
What was going on that prompted this change? Do we need to add a test case?

Reply:
Oh is this the same changeset as https://github.com/qiime2/q2-sample-classifier/pull/158? If so, you can address my questions there.

Reply:
yes sorry need to rebase (forgot I had pulled in those changes here), will respond over there